armi/settings/fwSettings/globalSettings.py

```diff
@@ -656,7 +656,7 @@ def defineSettings() -> List[setting.Setting]:
         ),
         setting.Setting(
             CONF_TRACK_ASSEMS,
-            default=True,
+            default=False,
             label="Save Discharged Assemblies",
             description="Track assemblies for detailed fuel histories. Disable in case "
```

Comment:
Can we add a comment here about what it means to "track" assemblies? Similar to what you say elsewhere about the assemblies going into the SFP or not.

Reply:
Fixed